### ISSUE TITLE
AMP URLs 404 after disabling plugin

### ISSUE THREAD

**Issue by liamzebedee** [TryGhost/Ghost#9712]
### Issue Summary

Enabling AMP on a site and then disabling it results in 404's for the ampproject.org redirects.

For example, my post at `https://liamz.co/blog/beers-trains-and-blockchains-hackatrain-2018/` was cached on ampproject.org as `https://liamz-co.cdn.ampproject.org/c/s/liamz.co/blog/beers-trains-and-blockchains-hackatrain-2018/amp/`. I then realised Ghost didn't convert `<video>` tags to `<amp-video>` (which makes sense since they're not supported just yet), so I turned off the AMP plugin. amproject.org then redirected users back to `https://liamz.co/blog/beers-trains-and-blockchains-hackatrain-2018/` with a trailing `/amp/`, which subsequently 404'd.

### To Reproduce

1. Enable AMP plugin
2. Write a post and publish it
3. Get it inserted into an AMP cache such as ampproject.org
4. Visit cached version
5. Disable AMP plugin
6. Wait until cache updates.
7. Visit AMP'ed URL and receive 404.

### Technical details:

* Ghost Version: 1.21.1
* Node Version: 8.11.3
* Browser/OS: Amazon Ubuntu
* Database: SQLite

**Comment by liamzebedee** [comment]
My current fix is this `redirects.json`

```
[
    {
        "from": "/(.*)\/amp",
        "to": "/$1/"
    }
]
```

**Comment by kevinansfield** [comment]
This was fixed in [1.21.6](https://github.com/TryGhost/Ghost/releases/tag/1.21.6). Please upgrade your version of Ghost 😄 current version is 1.22.7

**Comment by liamzebedee** [comment]
Derp!
